### ISSUE TITLE
Desktop mega menu unable to tab through sub menus

### ISSUE THREAD

**Issue by wullaski** [department-of-veterans-affairs/vets-website#21560]
## Describe the defect
When using the keyboard to navigate the mega menu the user is unable to tab through to any other sections besides the first one(healthcare).

Upper level items are getting a tabindex of -1 which explains why they can't be focused on with the keyboard. MVP fix could be to just remove the tabindex of -1

## To Reproduce
1. Go to va.gov
2. Use the tab key to focus on VA benefits and healthcare
3. Use the space key to open the menu
4. Attempt to use the tab key to focus on the second item 'Disability'

## AC / Expected behavior
The tab key should cycle through the upper level items and the space bar should let the user enter the menu for each one. Shift+Tab should take you back to the upper level items requiring the space bar to enter again. If the user tabs to the end of a sub menu area it should take the focus to the next upper level item.

## Screenshots
![Screen Shot 2022-07-05 at 11 59 06 AM](https://user-images.githubusercontent.com/2982977/177370501-692e0760-d197-49bc-92aa-2e62798ae6de.png)

![Screen Shot 2022-07-05 at 12 25 45 PM](https://user-images.githubusercontent.com/2982977/177373868-38500b77-7a13-4d47-a80d-7a3613b1baae.png)

 ## Additional context
This could be a part of a larger effort here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/32443

## Desktop (please complete the following information if relevant, or delete)
 - OS: Mac
 - Browser Firefox
 - Version 102

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [x] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`
